Modules/gcmodule.c

```diff
@@ -508,6 +526,7 @@ static PyMethodDef GcMethods[] = {
     GC_FREEZE_METHODDEF
     GC_UNFREEZE_METHODDEF
     GC_GET_FREEZE_COUNT_METHODDEF
+    GC__COLLECT_DELAYED_OBJECTS_METHODDEF
```

Comment:
Let's add a guard to this function so it's not exposed in normal builds

Reply:
Done! 

See
* https://github.com/python/cpython/pull/116238/files#r1509965460
* https://github.com/python/cpython/pull/116238/files#r1509965501

If the function is declared with ifdef macro, AC smartly exposes the method according to the condition. :)